src/components/ProjectNewView/flow/SelectBatches.tsx

```diff
@@ -129,17 +130,17 @@ export default function SelectBatches(props: SelectBatchesProps): JSX.Element |
     []
   );
 
-  const getSearchFields = useCallback(
-    (debouncedSearchTerm: string): FieldNodePayload[] => [
+  const getSearchFields = useCallback((debouncedSearchTerm: string): FieldNodePayload[] => {
+    const phraseMatchQuery = removeDoubleQuotes(debouncedSearchTerm);
+    return [
       {
         operation: 'field',
         field: 'batchNumber',
-        type: 'Fuzzy',
-        values: [debouncedSearchTerm],
+        type: phraseMatchQuery ? 'PhraseMatch' : 'Fuzzy',
+        values: [phraseMatchQuery || debouncedSearchTerm],
```

Comment:
Suggestion: create a util that takes the search term and type(s) as input and returns a `{ type, values }`
Util can do the remove double quotes, and return the type and values.
We could in other places import the util and spread the result of using it.
Number of changes will be about the same but logic will be centralized.